datafusion/proto/src/generated/datafusion.rs

```diff
@@ -0,0 +1,1512 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
I guess I do not need to commit this file, right?

Reply:
Correct -- it should not be part of the PR